File: web/src/model/victron_controller/dashboard/v0_2_0/from_0_1_0_actuated-i32.ts
// @ts-nocheck
import {ActuatedI32 as v0_1_0_ActuatedI32} from '../v0_1_0/ActuatedI32'
import {ActuatedI32 as v0_2_0_ActuatedI32} from './ActuatedI32'

export function convert__actuated_i32__from__0_1_0(from: v0_1_0_ActuatedI32): v0_2_0_ActuatedI32 {
    return new v0_2_0_ActuatedI32 (
        from.target_value,
        JSON.parse(JSON.stringify(from.target_owner)),
        JSON.parse(JSON.stringify(from.target_phase)),
        from.target_since_epoch_ms,
        JSON.parse(JSON.stringify(from.actual))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/v0_2_0/from_0_1_0_actuated_i32.rs


pub fn convert__actuated_i32__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::actuated_i32::ActuatedI32) -> crate::victron_controller::dashboard::v0_2_0::actuated_i32::ActuatedI32 {
    crate::victron_controller::dashboard::v0_2_0::actuated_i32::ActuatedI32 {
        target_value: from.target_value.clone(),
        target_owner: serde_json::from_value(serde_json::to_value(&from.target_owner).unwrap()).unwrap(),
        target_phase: serde_json::from_value(serde_json::to_value(&from.target_phase).unwrap()).unwrap(),
        target_since_epoch_ms: from.target_since_epoch_ms.clone(),
        actual: serde_json::from_value(serde_json::to_value(&from.actual).unwrap()).unwrap(),
    }
}